tests/helpers/diff_test_helpers.py

```diff
@@ -0,0 +1,147 @@
+import os
+import re
+import shutil
+import subprocess
+import sys
+from typing import Tuple
+
+from ethereum_spec_tools.forks import Hardfork
+
+
+def get_fork_paths(fork_name: str) -> Tuple:
+    """
+    Get the location of the fork represented by `fork_name`
+    as well as the previous fork
+    """
+
+    if fork_name == "frontier":
+        sys.exit("No fork before frontier")
+
+    forks = Hardfork.discover()
+
+    for i, fork in enumerate(forks):
+        if fork.short_name == fork_name:
+            return forks[i - 1].path[0], fork.path[0]  # type: ignore
+
+    return None, None
+
+
+def find_source_files(path: str) -> set:
+    """
+    Find .py files in a particular location
+    """
+    file_list = []
+    for directory, _, filenames in os.walk(path):
+        for filename in filenames:
+            if filename.endswith(".py"):
+                file_path = os.path.join(directory, filename)
+                file_path = file_path[len(path) + 1 :]
+                file_list.append(file_path)
+
+    return set(file_list)
+
+
+def process_source_code(path: str, name: str) -> None:
+    """
+    Clean up the source code.
+    Removes comments and empty lines.
+    """
+    if not os.path.isfile(path):
+        open(name, "a").close()
+        return
+
+    with open(path) as f:
+        code = f.read()
+
+    code = re.sub(re.compile('""".*?"""', re.DOTALL), "", code)
+    code = re.sub(re.compile("#.*?\n"), "\n", code)
+    code = re.sub(re.compile("\n\s*\n"), "\n", code)
```

Comment:
Haven't tested, but I think you can replace the `compile` with just a plain string:

```suggestion
    code = re.sub('""".*?"""', "", code, flags=re.DOTALL)
    code = re.sub("#.*?\n", "\n", code)
    code = re.sub("\n\s*\n", "\n", code)
```